### ISSUE TITLE
Placeholder for input on row adding

### ISSUE THREAD

**Issue by perfectstrong** [mbrn/material-table#810]
**Describe the bug**
Not really a bug but when the column header is a React Component, browser will warn whenever attempting to add a new row
````
Failed prop type: Invalid prop `placeholder` of type `object` supplied to `ForwardRef(TextField)`, expected `string`.
````

**To Reproduce**
````javascript
columns = [
        {
            field: "firstCol",
            title: <ReactComponent/>,
        },
        {
            field: "secondCol",
            title: "B"
        },
]
````

**Suggestion**
We should have a custom `placeholder` field for each column in case of having React Component as column header.

**Screenshots**
![image](https://user-images.githubusercontent.com/23375518/60896173-2acdb900-a266-11e9-9e38-36c73e2c0a7d.png)

**Comment by masbaehr** [comment]
Seems to be a duplicate, i provided a workaround in my issue thread, but it really needs a fix :/

https://github.com/mbrn/material-table/issues/1260

**Comment by aabuelenin** [comment]
here's the workaround I used:
```
components={{
                        EditField: props => {
                            const newProps = {
                                ...props,
                                ...{
                                    columnDef: {
                                        title: '',
                                    },
                                },
                            };
                            return <MTableEditField {...newProps} />;
}}
```
you can replace title: '' with props.columnDef.placeholder if you want

**Comment by Domino987** [comment]
Closed in favor of #1260
